admin/src/react-components/accounts.js

```diff
@@ -257,11 +258,11 @@ export const AccountList = withStyles(styles)(
 );
 
 export const AccountEdit = withStyles(styles)(props => {
-  const { classes } = props;
+  const { classes, ...other } = props;
 
   return (
-    <Edit {...props}>
-      <SimpleForm>
+    <Edit {...other}>
```

Comment:
Was passing `classes` through breaking something?

Reply:
Yes, was throwing an error in my console. Material ui was not liking it.

Reply:
Ah right, you mentioned that "Fixed some errors in console about styling".